File: app/src/main/java/br/pedroso/upcomingmovies/moviedetails/MovieDetailsView.java
package br.pedroso.upcomingmovies.moviedetails;

import java.util.List;

import br.pedroso.upcomingmovies.domain.Movie;

public interface MovieDetailsView {
    void renderMovieDetails(Movie movie);

    void hideSimilarMoviesPanel();

    void renderSimilarMovies(List<Movie> movies);

    void displaySimilarMoviesPanel();
}

File: app/src/main/kotlin/br/pedroso/upcomingmovies/moviedetails/MovieDetailsView.kt
package br.pedroso.upcomingmovies.moviedetails

import br.pedroso.upcomingmovies.domain.Movie

interface MovieDetailsView {
    fun renderMovieDetails(movie: Movie)

    fun hideSimilarMoviesPanel()

    fun renderSimilarMovies(movies: List<Movie>)

    fun displaySimilarMoviesPanel()

    fun startMovieDetailsActivity(id: Int)
}
